pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md

```diff
@@ -335,6 +336,35 @@ with this one:
 
 Save and exit the file.
 
+Now, if you choosed `Yes` when prompted for enabling SSL connection, get the CA certificate from the OVHcloud Control Panel:
```

Comment:
Now, if you choose `Yes` when prompted to enable SSL connection, get the CA certificate from the OVHcloud Control Panel:

Reply:
Done